interfaces/builtin/unity7.go

```diff
@@ -249,21 +249,21 @@ dbus (send)
 # dbusmenu
 dbus (send)
     bus=session
-    path=/MenuBar{,/[0-9]*}
+    path=/{MenuBar{,/[0-9A-F]*},com/canonical/menu/[0-9A-F]*}
```

Comment:
It is needed for Chrome to work so I included it here. I could do it as a separate PR if you prefer.